plugins/adr-backend/package.json

```diff
@@ -36,6 +36,9 @@
     "@backstage/integration": "workspace:^",
     "@backstage/plugin-adr-common": "workspace:^",
     "@backstage/plugin-search-common": "workspace:^",
+    "@types/express": "^4.17.15",
```

Comment:
@kuangp I have switched the ```@types/express``` to be a dev dependency, however it seems the linter is now telling me to switch it back (https://github.com/backstage/backstage/actions/runs/3788464841/jobs/6441277500#step:15:11).

Is there a reason it would be doing this?

Reply:
ah my apologies, it looks like since we are [exporting a type from the `express` package](https://github.com/backstage/backstage/pull/15348/files#diff-1b1b4bc67372dfafc27ee76b22e5b66f065e746eecc7a683133968e420623a3cR18), we also need to specify `@types/express` as an explicit dependency of this plugin

ref: [lint command](https://github.com/backstage/backstage/blob/master/packages/repo-tools/src/commands/type-deps/type-deps.ts), [reasoning](https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies)

Reply:
That makes sense. 

I have adjusted the dependencies back and CI is now passing